interactive-clap-derive/docs/clap_enum_for_named_arg_docstring.md

```diff
@@ -0,0 +1,27 @@
+derive of helper enum for structs with `#[interactive_clap(named_arg)]` on fields
```

Comment:
It's internal doc, so I would better keep it with the code at the level with the code

Reply:
assumed this to be the duplicate of https://github.com/near-cli-rs/interactive-clap/pull/26#discussion_r1946533737